### PR TITLE
Version with seatmap and info message types

### PR DESCRIPTION
**Pull request by VjeraTurk**
## Why this should be merged
To support the new Message Types already published in the Protocol (there's no dev branch)

- Create bot version 7.0.0. It should support new services from the CMP release-7:
  - `CountryEntryRequirementsService`
  - `SeatMapService`
  - `SeatMapAvailabilityService`
- Update the example configs to support new services.
- Update mockup partner plugin to support new services.

## How this works
New MessageTypes were added
`SeatMapRequest`
`SeatMapResponse`
`SeatMapAvailabilityRequest`
`SeatMapAvailabilityResponse`
`CountryEntryRequirementsRequest`  
`CountryEntryRequirementsResponse`

There were changes to 
`MintResponse` `MintId` and  `ValidationResponse` `ValidationId`   
from type `string`  to `UUID`

Fixes in protocol were required: new protocol version published - release-8
`grpc` package version increased, deprecated function `Dail` exchanged for a new one `NewClient`

## How this was tested

With API calls to partner plugin mockup from Postman using grpc reflection

### CODE REVIEW

**Comment by VjeraTurk** [comment]
@havan is there a use-case documented somewhere that I could use to mock the data more accurately to what the actual use case may be?

**Comment by havan** [comment]
@VjeraTurk please ask @SamJaarsma in Discord, he had some example messages. They are not for CMP but you can see what a real data looks like and use it for mocks.

**Comment by VjeraTurk** [comment]
I need to increase the protocol version and address the changes when fixes are applied

**Comment by SamJaarsma** [comment]
> I need to increase the protocol version and address the changes when fixes are applied

Yes, let's see if we can call this a 7.0.1 or 7.1.0

**Comment by VjeraTurk** [comment]
> > I need to increase the protocol version and address the changes when fixes are applied
> 
> Yes, let's see if we can call this a 7.0.1 or 7.1.0

Just to have it in written form - on yesterday's daily-sync @Noctunus said it's ok to respect the initial idea and push the bot from v6.1.0 straight to v8.0.0.

**Comment by SamJaarsma** [comment]
> > > I need to increase the protocol version and address the changes when fixes are applied
> > 
> > 
> > Yes, let's see if we can call this a 7.0.1 or 7.1.0
> 
> Just to have it in written form - on yesterday's daily-sync @Noctunus said it's ok to respect the initial idea and push the bot from v6.1.0 straight to v8.0.0.

Confirmed - message types updates are always integers, so we had released V7, so your update is V8 and conclusively the bot has to be v8.0.0

File: internal/messaging/service.go [hunk 1]
```diff
@@ -34,6 +37,7 @@ var (
 	_ Service = (*partnerService)(nil)
 	_ Service = (*pingService)(nil)
 	_ Service = (*transportService)(nil)
+	_ Service = (*seatMapService)(nil)
 )
```
Comment (havan) [inline internal/messaging/service.go]: Don't we need the other two services here?

File: internal/rpc/client/client.go [hunk 2]
```diff
@@ -48,7 +48,7 @@ func (rc *RPCClient) Start() error {
 		opts = append(opts, grpc.WithTransportCredentials(tlsCreds))
 	}
 
-	cc, err := grpc.Dial(fmt.Sprintf("%s:%d", rc.cfg.Host, rc.cfg.Port), opts...)
+	cc, err := grpc.NewClient(fmt.Sprintf("%s:%d", rc.cfg.Host, rc.cfg.Port), opts...)
```
Comment (havan) [inline internal/rpc/client/client.go:51]: Why this change? New lib version?

File: internal/messaging/response_handler.go [hunk 3]
```diff
@@ -503,7 +503,7 @@ func createTokenURIforMintResponse(mintResponse *bookv1alpha.MintResponse) (stri
 	attributes := []Attribute{
 		{
 			TraitType: "Mint ID",
-			Value:     mintResponse.GetMintId(),
+			Value:     mintResponse.GetMintId().Value,
```
Comment (havan) [inline internal/messaging/response_handler.go:506]: This is because the change of mint id from string to UUID type right?

File: internal/messaging/service.go (hunk 1 above)
Comment (VjeraTurk) [reply internal/messaging/service.go]: Added

File: internal/rpc/client/client.go (hunk 2 above)
Comment (VjeraTurk) [reply internal/rpc/client/client.go:51]: Yes,
the upgrade form v1.62.2 -> v1.64.1
 Dail is deprecated, remarks to use NewClient instead
https://pkg.go.dev/google.golang.org/grpc@v1.64.1#Dial
https://pkg.go.dev/google.golang.org/grpc@v1.64.1#NewClient

File: internal/messaging/response_handler.go (hunk 3 above)
Comment (VjeraTurk) [reply internal/messaging/response_handler.go:506]: Yes, should I test this part just in case?

File: internal/rpc/client/client.go (hunk 2 above)
Comment (VjeraTurk) [reply internal/rpc/client/client.go:51]: I tried to avoid using `go mod tidy` or `go get -u` so I'm not sure how this package got upgraded - perhaps as one of the dependencies?

File: internal/messaging/service.go [hunk 4]
```diff
@@ -40,6 +40,7 @@ var (
 	_ Service = (*seatMapService)(nil)
 	_ Service = (*seatMapAvailabilityService)(nil)
 	_ Service = (*seatMapAvailabilityService)(nil)
```
Comment (SamJaarsma) [inline internal/messaging/service.go]: is seatMapAvailabilityService duplicated?

Comment (VjeraTurk) [reply internal/messaging/service.go]: removed

File: internal/messaging/response_handler.go (hunk 3 above)
Comment (havan) [reply internal/messaging/response_handler.go:506]: It's fine

File: internal/rpc/client/client.go (hunk 2 above)
Comment (havan) [reply internal/rpc/client/client.go:51]: It looks fine to me If the new versions doesn't break anything.

**Review by havan — APPROVED** [review]
